client/packages/requisitions/src/ResponseRequisition/api/hooks/line/useDeleteResponseLines.ts

```diff
@@ -4,32 +4,57 @@ import {
   useTranslation,
   useTableStore,
   useDeleteConfirmation,
+  noOtherVariants,
 } from '@openmsupply-client/common';
 import { useResponse } from '..';
 import { useResponseNumber } from '../document/useResponse';
 import { useResponseApi } from '../utils/useResponseApi';
 import { useResponseLines } from './useResponseLines';
+import { useResponseRequisitionLineErrorContext } from '../../../context';
 
 export const useDeleteResponseLines = () => {
-  const { lines } = useResponseLines();
+  const t = useTranslation();
+  const queryClient = useQueryClient();
   const api = useResponseApi();
   const requestNumber = useResponseNumber();
+  const { lines } = useResponseLines();
   const isDisabled = useResponse.utils.isDisabled();
-  const queryClient = useQueryClient();
   const { mutateAsync } = useMutation(api.deleteLines, {
     onSettled: () =>
       queryClient.invalidateQueries(api.keys.detail(requestNumber)),
   });
-  const t = useTranslation();
+  const errorsContext = useResponseRequisitionLineErrorContext();
 
   const selectedRows = useTableStore(state =>
     lines.filter(({ id }) => state.rowState[id]?.isSelected)
   );
 
   const onDelete = async () => {
-    mutateAsync(selectedRows).catch(err => {
+    let result = await mutateAsync(selectedRows).catch(err => {
       throw err;
```

Comment:
lol not your changes but... if we're just throwing again, why are we catching this error?

Reply:
Can just console it